### PR TITLE
[1] feat(twap): display orders grouping

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

image

 1. Hide "Avg. execution price" from parent receipt modal
 2. Keep showing parent orders on Open orders tab
 3. Shift chidren orders to display grouping
 4. Hide status box in parent parent when its not in Signing state
 5. Don't display parent orders in history

  # To Test

Please, test through https://github.com/cowprotocol/cowswap/pull/2876

### CODE REVIEW

File: src/api/gnosisProtocol/hooks.ts [hunk 1]
```diff
@@ -34,8 +30,7 @@ type TwapPartOrdersMap = { [twapOrderHash: string]: TwapPartOrderItem }
  */
 export function useGpOrders(account?: string | null, refreshInterval?: number): OrderWithComposableCowInfo[] {
   const { chainId } = useWalletInfo()
-  const emulatedTwapOrders = useAtomValue(emulatedTwapOrdersAtom)
-  const twapParticleOrders = useAtomValue(twapPartOrdersListAtom)
+  const twapPartOrders = useAtomValue(twapPartOrdersAtom)
```
Comment (shoom3301) [inline src/api/gnosisProtocol/hooks.ts:33]: Skip changes in this file, just look into https://github.com/cowprotocol/cowswap/pull/2876

File: src/legacy/state/orders/updaters/GpOrdersUpdater.ts [hunk 2]
```diff
@@ -100,7 +91,7 @@ function _transformGpOrderToStoreOrder(
     status,
     receiver: receiver || '',
     apiAdditionalInfo: order,
-    isCancelling,
+    isCancelling: composableCowInfo?.isCancelling || (apiStatus === 'pending' && order.invalidated), // already cancelled in the API, not yet in the UI
```
Comment (shoom3301) [inline src/legacy/state/orders/updaters/GpOrdersUpdater.ts:94]: Also skip, changed in https://github.com/cowprotocol/cowswap/pull/2876

File: src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx [hunk 3]
```diff
@@ -177,8 +178,17 @@ export function OrderRow({
...
+  const isChildOrder = getIsComposableCowChildOrder(order)
+  const isComposableCowParentOrder = getIsComposableCowParentOrder(order)
+  const isStatusBoxHidden = isComposableCowParentOrder && order.status !== OrderStatus.PRESIGNATURE_PENDING
+
   return (
-    <TableRow data-id={order.id} isOpenOrdersTab={isOpenOrdersTab} isRowSelectable={isRowSelectable}>
+    <TableRow
+      data-id={order.id}
+      isChildOrder={isOpenOrdersTab && isChildOrder}
+      isOpenOrdersTab={isOpenOrdersTab}
```
Comment (alfetopito) [inline src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx:189]: Shouldn't the `isChildOrder` be passed as is and on `TableRow` you check whether it's in the open orders tab?

**Review by alfetopito — APPROVED** [review]

**Review by elena-zh — APPROVED** [review]
LGTM in #2876

**Pull request merged**